static/js/collapsible-filters/interactions.js

```diff
@@ -123,6 +123,18 @@ export default class Interactions {
       eventType: 'update',
       storageKey: 'query',
       callback: () => {
+        const repeatedSearchTriggers = [
```

Comment:
What is a 'repeated' search? I'm a bit confused by that terminology.

Reply:
it's any search that wasn't triggered by typing in a query into the searchbar, I can update!